server/models/announcementModel.js

```diff
@@ -0,0 +1,19 @@
+const Sequelize = require('sequelize');
+const connection = require('../config/db');
+
+const Annoucement = connection.define('user', {
+  announcement: {
+    type: Sequelize.VARCHAR,
+  },
+  cohortId: {
+    type: Sequelize.INTEGER,
+  },
+  timeCreated: {
```

Comment:
When you insert something into the table, sequelize automatically adds createdAt and lastUpdatedAt columns. So there's no need to create them:D!